resource/ui/MainWindow.ui

```diff
@@ -141,7 +141,7 @@
               <object class="GtkLabel" id="start_in_tray_label">
                 <property name="visible">True</property>
                 <property name="can-focus">False</property>
-                <property name="label" translatable="yes">Start In Tray</property>
+                <property name="label" translatable="yes">           Start In Tray           </property>
```

Comment:
Why? Please don't use spaces at the beginning and end, if needed use the appropriate policies.